generators/next-js/templates/base/.eslintrc.json

```diff
@@ -7,12 +7,13 @@
     "env": {
         "browser": true
     },
+    "ignorePatterns": ["next.config.js"],
```

Comment:
You should instead add an `overrides` for this files to disable rules that doesn't apply instead of completely disabling linting.

Reply:
@jvasseur you motivated me to close this one. 

The issue I try to avoid with ignorePattern is the following :

 `Parsing error: "parserOptions.project" has been set for u/typescript-eslint/parser.` is mainly caused (if I understand correctly) because esLint includes files that my tsconfig.json doesn't include. 
 
 Include this specific file in the `tsconfig.json` to enable the ts type check works that being said

Reply:
Not 100% sure it would work but what about resetting the parser to the default one for this file with something like :
```json
{
    "overrides": [
        {
            "files": ["next.config.js"],
            "parser": "espree"
        }
    ]
}
```

Reply:
works like a charm 👍